app/src/androidTest/java/com/protonvpn/actions/SettingsRobot.java

```diff
@@ -65,7 +65,7 @@ public SettingsRobot toggleOffSplitTunneling() {
         return this;
     }
 
-    public SettingsRobot openExcluededIPAddressesList() {
+    public SettingsRobot openExcludedIPAddressesList() {
```

Comment:
If this is a public api, then it might not be the kind of thing to change...

Reply:
This is fine to change